midi_app_controller/models/binds.py

```diff
@@ -1,5 +1,6 @@
 from typing import Optional
-from pydantic import BaseModel, Field, root_validator
+from pydantic import BaseModel, Field
+from pydantic_compat import model_validator
```

Comment:
It's for `v1` of `pydantic`. In `v2` the `model_validator` should be imported from `pydantic`.